attendance.php

```diff
@@ -957,6 +980,39 @@ function recordChange(ele, event_id, student_id, present) {
 	.main-attendance-view tbody tr:nth-child(2n + 1) {
 		background-color: #eee;
 	}
+
+	#save-button-holder input{
+	    background: #eb3b45;
+	    -webkit-border-radius: 5px;
```

Comment:
it doesn't hurt to have these, but the prefixed border radius things haven't been necessary for a long time now

Reply:
Yeah -- I saw you were using them in your website code so did it here as well.